src/core/Document.js

```diff
@@ -0,0 +1,29 @@
+class Document {
+  constructor(kuzzle, response) {
+    this._id = response.result._id;
+    this._source = response.result._source;
+    this._index = response.index;
+    this._collection = response.collection;
+    this._kuzzle = kuzzle;
+  }
+
+  subscribe () {
```

Comment:
:trollface: https://youtu.be/dv13gl0a-FA?t=62
https://github.com/kuzzleio/sdk-javascript/blob/2cd4220cc06548254590c8d1cbbb3f79eaccd89b/src/Document.js#L312

Reply:
Lol, we can do better than that now ;-)